src/Workspaces/Core/Portable/Utilities/Operations/IOperationScope.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.Utilities
+{
+    /// <summary>
+    /// Represents a single scope of a context of executing potentially long running operation. Scopes allow multiple
+    /// components running within an operation to share the same context.
+    /// </summary>
+    public interface IOperationScope : IDisposable
+    {
+        /// <summary>
+        /// Gets user readable operation description.
+        /// </summary>
+        string Description { get; set; }
```

Comment:
>set; [](start = 34, length = 4)

This is mutable?

Reply:
yes.  intentiaonlly.  this is how some operaiton updates the description that is being shown in the UI as progress is happening.

Reply:
Why is that not part of ProgressInfo?

---
In reply to: [551563044](https://github.com/dotnet/roslyn/pull/50211#discussion_r551563044) [](ancestors = 551563044)